README.md

```diff
@@ -150,7 +150,7 @@ We have several feature flags currently in use. Their purposes are as follows:
 
 * `displayIQEngineGitHub`: When true this flag shows the link for the IQ Engine GitHub repo
 
-* `hideInternalBranding`: When true this flag hides an additional logo for customers using IQ Engine
+* `displayInternalBranding`: When true this flag displays an additional logo for customers using IQ Engine
```

Comment:
Might make sense to call out this will be false by default and that the others are true maybe ass a default after each one.